apps/services/routers.py

```diff
@@ -100,49 +101,47 @@ async def read_user_request_list(
 
 
 @services_router.post(
-    "/{university_id}/user-request/",
-    name="create_user_request",
-    response_model=JSENDOutSchema[CreateUserRequestOut],
-    summary="Create user request",
-    responses={200: {"description": "Successful create user request response"}},
-    tags=["Services application"],
+    "/{university_id}/create_request_for_hostel_accommodation/",
```

Comment:
change url pls, see alrady existing ednpoint's url for hostel accommodation

Reply:
ok